apps/aecore/src/aec_tx_pool.erl

```diff
@@ -151,15 +155,19 @@ push(Tx) ->
     push(Tx, tx_created).
 
 -spec push(aetx_sign:signed_tx(), event()) -> ok | {error, atom()}.
-push(Tx, Event = tx_received) ->
+push(Tx, Event) ->
+    push(Tx, Event, ?DEFAULT_PUSH_TIMEOUT).
+
+-spec push(aetx_sign:signed_tx(), event(), push_timeout()) -> ok | {error, atom()}.
+push(Tx, Event = tx_received, Timeout) ->
```

Comment:
These 5 secs are what was there before but hidden as implicit timeout of a gen_server call. 
Now it's just configurable, for clients requesting this server.

It seems long, but it's not rare to see stratum CT test fail because TX is still sitting in mempool.